content/v2.0/influx-templates/create.md

```diff
@@ -1,21 +1,105 @@
 ---
-title: Create an Influx Template
+title: Create an Influx template
 description: >
-  placeholder
+  Use the InfluxDB UI and the `influx pkg export` command to create Influx templates.
 menu:
   v2_0:
     parent: Influx templates
     name: Create a template
-    identifier: Create an Influx Template
+    identifier: Create an Influx template
 weight: 101
 v2.0/tags: [influx templates]
 ---
 
-## Placeholder outline
-- Create a template
-  - `influx pkg`
-  - Package everything vs package specific things
-  - Include secrets (only keys are included, uses will still need to populate the secret)
-  - Submit a template to the InfluxDB community template repo
-  - View your package summary
-  - Validate a package
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create Influx templates.
+Add buckets, Telegraf configurations, tasks, and more in the InfluxDB UI and export
+them as a template.
+
+{{% note %}}
+The simplest way to create a template is to create a new organization and build
+the template within the organization.
+**InfluxDB OSS** supports multiple organizations.
+In **InfluxDB Cloud**, your user account is an organization.
+**We recommend using InfluxDB OSS to create Influx templates.**
```

Comment:
can folks create a template in Cloud?

Reply:
They can but it's not recommended. The workflow (currently) just isn't great for creating templates in cloud.